files/default/ckan-monitor-job-queue.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+QUEUE=$(/usr/lib/ckan/default/bin/paster --plugin=ckan jobs list -c /etc/ckan/default/production.ini 2>/dev/null)
+if [ "$QUEUE" != "" ]; then
+    NOW=$(date +'%Y-%m-%dT%H:%M:%S')
+    echo "$QUEUE" > /var/log/ckan/ckan-worker-queue.$NOW.log
```

Comment:
how does this get cleaned up? 

can you it go into the same log with timestamps pre and post pull as well as a count of the items at the end with queue depth time from oldest to current time

Reply:
The other pull request adds log archival for everything under `/var/log/ckan`, which should handle cleanup. I'm not sure about appending to one log file, because I'm not certain about exactly how Cloudwatch works, but it's not hard to do.

Reply:
CloudWatch metrics now include queue size and maximum job age (in seconds)